generator/internal/api/model.go

```diff
@@ -524,6 +532,10 @@ func (field *Field) DocumentAsRequired() bool {
 	return slices.Contains(field.Behavior, FIELD_BEHAVIOR_REQUIRED)
 }
 
+func (f *Field) Singular() bool {
```

Comment:
Call this in a test? Or save for a follow up?

Reply:
Saving for a followup, with that we can write `{{#Singular}}` in the mustache templates.